### ISSUE TITLE
Atmospheric alert console reports problems in... SPACE!

### ISSUE THREAD

**Issue by LandoTheClimber** [NTStation/NTstation13#850]
Revision: 2014-07-19 a092177c6b54caa824068eb48f295c848ffb1639
Problem Description: Theres an atmospheric alert in space
What did you expect to happen? There wouldn't be cause its space...
What happened instead? THERE WAS ONE
Why is this bad/What are the consequences? Well it doesn't make any fucking sense why would the alert console care
Steps to reproduce the problem: Just start a round and check the atmos alert console (with moonoutpost19.dmm loaded I think)
Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?): 
![ss 2014-07-19 at 03 04 47](https://cloud.githubusercontent.com/assets/2679005/3635508/972392fa-0f77-11e4-81bf-0c410b184cd5.png)
oh and also the away mission is moonoutpost19.dmm and when I jump to area space it takes me there so i think its an issue with that..

**Comment by WJohn** [comment]
Interesting, I'm not exactly sure why it can pick up air alarms off the z level... should probably have it so it can only recognize those on its own.

**Comment by LandoTheClimber** [comment]
Yeah it should, since in addition to this every round there is always a minor atmos alert at the gulag

**Comment by ike709** [comment]
Last time I tried to make something work on only 1 z level people got pissed.

**Comment by LandoTheClimber** [comment]
do it anyways :+1:

**Comment by ike709** [comment]
But what about the mining station?

**Comment by LandoTheClimber** [comment]
What about just stopping the console from seeing alarms on the away level?

**Comment by Tokiko1** [comment]
I noticed this bug when the server just updated. Odd thing is that during that time, the away mission was set to clownplanet for a few rounds before being changed to moon outpost and yet this bug still occured during those clownplanet rounds.

**Comment by ike709** [comment]
@Tokiko1 The first rounds after the update SoS was having issues because he wasn't entirely sure on how to make the away mission work and disable the clown planet.
